dev/bots/analyze.dart

```diff
@@ -507,11 +508,112 @@ Future<void> _verifyNoBadImportsInFlutterTools(String workingDirectory) async {
   }
 }
 
+Iterable<File> dartFiles(String workingDirectory) sync* {
+  yield* Directory(path.join(workingDirectory, 'packages'))
+    .listSync(recursive: true)
+    .whereType<File>()
+    .where((File file) => path.extension(file.path) == '.dart');
+  yield* Directory(path.join(workingDirectory, 'examples'))
+    .listSync(recursive: true)
+    .whereType<File>()
+    .where((File file) => path.extension(file.path) == '.dart');
+  yield* Directory(path.join(workingDirectory, 'dev'))
+    .listSync(recursive: true)
+    .whereType<File>()
+    .where((File file) => path.extension(file.path) == '.dart');
+}
+
+final RegExp _findDeprecationPattern = RegExp(r'@[Dd]eprecated');
```

Comment:
Should/can we outlaw the lower-case `@deprecated`? I think you should always provide a textual string that explains what to use instead.

Reply:
I see we are essentially doing this in line 558/559. Never mind, then.